modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -200,7 +200,7 @@ $(document).ready(function() {
 
         let $target = $(event.currentTarget);
         let tabName = '';
-        let $tabPanel = $target.attr('data-target');
+        let $tabPanel = $($target).attr('data-target');
```

Comment:
Don't need to wrap $target, only $tabPanel :) Sorry, @ayumihamsaki ))

Reply:
No problem wasn't sure on that.